src/objdump.main.v3

```diff
@@ -59,5 +62,88 @@ def parseAndDump(engine: Engine, path: string) -> int {
 		if (f.imp == null) OUT.put1(" { %d bytes }", f.orig_bytecode.length);
 		OUT.outln();
 	}
+	for (i < module.heaptypes.length) {
+		var h = module.heaptypes[i];
+		OUT.put1("heaptypes[%d] = ", i);
+		h.render(OUT);
+		var fh = h.getFirstSuperType();
+		if (fh != null) fh.render(OUT);
+		OUT.outln();
+	}
+	for (i < module.abstypes.length) {
+		var a = module.abstypes[i];
+		OUT.put1("abstypes[%d] = ", i);
+		if (a.isAssignableTo(ValueType.BOTTOM)) {
+			OUT.put1("%q ", ValueType.BOTTOM.render);
```

Comment:
You should be able to just use the render method of `a`.

Reply:
I'm a bit confused. Where can we find the render method of a? I checked AbsTypeDecl class and Decl class but didn't find anything related.